src/sass/modules/modals.scss

```diff
@@ -1,4 +1,6 @@
 .modal {
+  background-color: $white;
+  border-radius: 6px;
```

Comment:
we should maybe check with mark on this, but I think 6px is a copy over from mainapp.  this should likely be 4 or 8